### PR TITLE
WRQ-30274: Prepare for merging `feature/enact_v5` to `develop` branch

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We will merge `feature/enact_v5` branch to `develop` to release next major alpha version of enact.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Merged `feature/enact_v5` branch to `develop`
- Modified `npm-shrinkwrap.json` in sampler
- Changed ubuntu dist in travis.yml from `focal` to `jammy`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-30274

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Review by seunghoh — CHANGES_REQUESTED** [review]
Please fill the PR description when you make a PR

File: packages/sampler/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact Sampler, newest changes
 
 No significant changes.
 
+## [5.0.0-alpha.1] - 2024-07-11
+
+No significant changes.
```
Comment (MikyungKim) [inline packages/sampler/CHANGELOG.md:11]: I guess we missed the change log for this.
Could you add below?
```suggestion
### Changed

- `sampler` look and feel by updating to Storybook 8
```

Comment (vJIYEv) [reply packages/sampler/CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,35 @@ The following is a curated list of changes in the Enact project, newest changes
...
 
+## [5.0.0-alpha.1] - 2024-07-11
+
+### Added
+
+- `spotlight/SpotlightRootDecorator` config `rootId` to specify React DOM tree root for global event handlers
+
+### Changed
+
+- `ui/Marquee.MarqueeDecorator` to use `@enact/i18n/utils.isRtlText` API to determine the text directionality
```
Comment (MikyungKim) [inline CHANGELOG.md:22]:
```suggestion
- `ui/Marquee.MarqueeDecorator` to use `@enact/i18n/utils.isRtlText` API to determine the text directionality
- `sampler` look and feel by updating to Storybook 8
```

Comment (vJIYEv) [reply CHANGELOG.md:22]: ok

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Review by 0x64 — APPROVED** [review]

**Pull request merged**